src/magicbox/ResultPreviewsManager.ts

```diff
@@ -78,6 +84,14 @@ export class ResultPreviewsManager {
   }
 
   public async displaySearchResultPreviewsForSuggestion(suggestion: HTMLElement) {
+    const externalOptions = this.getExternalOptions();
+    const currentDelay = (this.lastDelay = Utils.resolveAfter(
+      externalOptions.delayBeforePopulate == undefined ? 200 : externalOptions.delayBeforePopulate
```

Comment:
I'd use `Utils.isNullOrUndefined()` here.

Reply:
This is exactly what I was looking for, thanks!